validation/omop52csv/tables.go

```diff
@@ -266,7 +266,7 @@ var (
 			"PERSON_ID":                      integer(true),
 			"DEVICE_CONCEPT_ID":              integer(true),
 			"DEVICE_EXPOSURE_START_DATE":     date(true),
-			"DEVICE_EXPOSURE_START_DATETIME": datetime(true),
+			"DEVICE_EXPOSURE_START_DATETIME": datetime(false),
```

Comment:
This change looks ok as `DEVICE_EXPOSURE.START_DATETIME` column is marked as `NULL` in OMOP DDL.

The only thing is to make sure we update validators / schema definitions to allow nullable values for this column in ash-codebase, @ajinkyagadewar

Reply:
Noted.